### DISCUSSION TITLE
on PC related platforms allow the file selector to intake text input

### DISCUSSION THREAD

**Discussion by Neeko-iko** [ppy/osu#18256] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/45524401/168407895-4c4e4451-f4b5-476d-a70b-cb6dd2a179bd.png)
i like the simplistic nature of how this looks, however i always find it so much easier to quickly toss in a filepath i've grabbed from my default file explorer.  either discretely allowing for random CTRL+Vs or a more clear text input button would be nice.

**Comment by peppy** [comment, marked as answer]
For sure, I'd also hope to be able to drag a folder into the game window to target it.
